aicsimageio/vendor/omexml.py

```diff
@@ -1255,7 +1255,7 @@ def get_Description(self):
             return get_text(description)
 
         def set_Description(self, text):
-            make_text_node(self.node, NS_SPW, "Description", test)
+            make_text_node(self.node, self.ns['spw'], "Description", text)
```

Comment:
I think you just found a bug in OMEXML and randomly fixed it?

Reply:
LOL, followed the commit message to the bioformats merge. Makes me think we should really be pulling in their omexml object every so often to update ours. Or use a git submodule?

Reply:
The problem is that this version of the file has lots of mods.  But this was clearly a big bug in the file that I just noticed and thought it was cheap to throw it in.

Reply:
could we create a repo of our own for this that pulls from the original repo and then has a branch that integrates the diffs of what's been added so that we have a path to update it via 1 submodule PR that we would reapply every time we have to update.